src/main/java/uk/ac/ebi/embl/api/entry/location/Location.java

```diff
@@ -20,6 +20,10 @@ public abstract class Location extends AbstractLocation implements Serializable
   private Long beginPosition;
   private Long endPosition;
 
+  private boolean threePrime;
```

Comment:
We should have the partiality in one place only or they may have conflicting values. If we have getters and setters in both CompoundLocation and Location then CompoundLocation should call the Location getters and setters and not have partiality fields itself. In this case CompoundLocation partiality setting should fail if there are no Locations yet.